lib/features/mailbox_dashboard/data/datasource_impl/session_storage_composer_datasoure_impl.dart

```diff
@@ -44,4 +46,14 @@ class SessionStorageComposerDatasourceImpl
       throw SaveToWebSessionFailException(errorMessage: e.toString());
     }
   }
+  
+  @override
+  void markComposerCacheIdentityOnWeb(AccountId accountId, UserName userName) {
+    final cache = getComposerCacheOnWeb();
+    final json = cache.toJson();
+    json['accountId'] = accountId.asString;
+    json['userName'] = userName.value;
+    final encoded = jsonEncode(json);
+    html.window.sessionStorage[EmailActionType.reopenComposerBrowser.name] = encoded;
```

Comment:
IMO, we should use `accountId` & `userName` as `key` in session storage and not as properties of json object.

Reply:
If that's the case, I would have to call 2 get method for composer reload cache & composer log out cache?

Reply:
Why do we need that? Why not have it together as one?

Reply:
You mean we can combine `reload page` & `session expired` into one and save to cache